src/cmd.rs

```diff
@@ -20,12 +20,31 @@ pub(crate) struct Cmd {
     /// Command name. E.g. if this is `"cmd"`, `/cmd ...` will call this command.
     pub(crate) name: &'static str,
 
-    // Command help message. Shown in `/help`.
-    // pub(crate) help: &'static str,
+    /// Command description.
+    pub(crate) description: &'static str,
+
+    /// Command usage. Shown in `/help <command>` or when the command itself is misused.
+    pub(crate) usage_args: &'static str,
```

Comment:
I think this should be a part of the description, for two reasons: (1) it saves users a command (2) currently it's not possible for a user to know that they need to run `/help cmd` to get argument info, they need to discover it somehow. If we show arguments in `/help` the UX will be slightly better IMO.

Reply:
What description format would you suggest?

This format can get quite long,
`command - description. Usage: /command args`

Reply:
I don't find your examples too long, but maybe we can add a new line before the usage, something like:

```
away - Set and remove away status.
       Usage: /away [MESSAGE]
```

Feel free to suggest a different formatting.

I also wonder if we should give more details in the description. For example, `/away` unsets the away status, `/away <message>` sets it. Current description doesn't say this explicitly, and it's hard for a user to figure this out from this description.